app/services/classification/values_hash_query.rb

```diff
@@ -69,11 +69,11 @@ def split_matcher(lookup_value)
         matcher_key = ''
         matcher_value = ''
 
-        lookup_value.match.split(LookupKey::KEY_DELM).each do |match_key|
-          element = match_key.split(LookupKey::EQ_DELM).first
-          matcher_key += element + ','
-          if LookupKey::MATCHERS_INHERITANCE.include?(element)
-            matcher_value = match_key.split(LookupKey::EQ_DELM).last
+        lookup_value.match.split(LookupKey::KEY_DELM).each do |match_keyval|
```

Comment:
Are these changes related?

Reply:
no, they are probably not, the tests for them just started to fail with my changes. I forgot I've not split them.
Going to do so now ^_^